### ISSUE TITLE
Are all datasets belonging to the mentioned INSPIRE themes also HVD?

### ISSUE THREAD

**Issue by JohannaOtt** [INSPIRE-MIF/hvd-inspire#4]
Hi everybody,

I am looking for a statement clarifying if
- all datasets belonging to the INSPIRE themes listed [here](https://inspire-geoportal.ec.europa.eu/srv/eng/catalog.search#/hvdshome?tab1=tab1) are HVD or
- in case a dataset belongs to one of these INSPIRE themes, the provider needs to check if it falls under HVD

Some background looking at the example of INSPIRE LU consisting of ELU and PLU:
Reading the [Commission Implementing Regulation (EU) 2023/138 of 21 December 2022 laying down a list of specific high-value datasets](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32023R0138&qid=1717140165407), it states in Annex 2.1:

> The earth observation and environmental category includes earth observation, including space-based or remotely-sensed data, as well as ground-based or in situ data, environmental and climate datasets within the scope of the INSPIRE data themes listed in the first table below and defined in Annexes I-III to Directive 2007/2/EC, and datasets produced or generated in the context of the legal acts listed in the second table below. The most up-to-date datasets as well as historical versions of datasets available in machine-readable format at all levels of generalisation available up to the scale of 1:5 000 covering the entire Member State when combined are included. If datasets are not available at this scale, but are available at higher spatial resolution(s)(13), they shall be provided at the available spatial resolution.

IMO, the only category PLU could fall under would be ground-based data but in order to clearly decide that, a more detailed definition of ground-based data would be needed. Reading the description, I could imagine very well that ELU data falls under HVD whereas PLU does not.

Any thoughts on this? Any hints on documents where this is clearly defined/distinguished?

Best, Johanna

**Comment by JohannaOtt** [comment]
@alexanderkotsev Do you know who I could reach out to in order to get feedback on this topic?

**Comment by JohannaOtt** [comment]
@fabiovinci Do you have any insights on who could comment on this?

**Comment by JohannaOtt** [comment]
@hogredan How is Germany legally interpreting the IR? In all German documentation I read, there is the same list provided as mentioned in the issue description but I cannot find any clear definition anywhere which of the two options I mentioned in the description is true.
Or do you maybe have an idea who could comment on the details?

**Comment by JohannaOtt** [comment]
@iuriemaxim Do you know who could help here? I think it would be important to align on that question in the community.

**Comment by iuriemaxim** [comment]
@JohannaOtt 

Hi,

My understanding is the following:

The text “... The most up-to-date datasets as well as historical versions of datasets…” from annex 2.1 of the [Commission Implementing Regulation (EU) 2023/138](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32023R0138&qid=1717140165407) reffers to all INSPIRE Data Themes that are falling under “Earth observation and environment”, including “Land Use”.

In article 2 (2) of the IR 2023/138 it is clearly stated that the definitions from INSPIRE Directive shall apply, namely for LU:

“Territory characterised according to its current and future planned functional dimension or socio-economic purpose (e.g. residential, industrial, commercial, agricultural, forestry, recreational).”

Therefore both ELU and PLU datasets falls under HVD.

Even more, PLU has also historical versions and would be usefull to compare PLU 2010 with PLU 2020.

So I do not understand the logic behind stating that PLU does not fall under HVD.

I do not see any article to sustain the statment “in case a dataset belongs to one of these INSPIRE themes, the provider needs to check if it falls under HVD" So I do not think that such a statment can be correct.

The INSPIRE Data Themes which are falling under Earth Observation and Environment HVD are the ones listed in the section 2.1 of the annex of Commission Implementing Regulation (EU) 2023/138](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32023R0138&qid=1717140165407) 
with their definitions as in the INSPIRE Directive.

As for the Geospatial HVD, the INSPIRE Data Themes are listed in section 1.1 of the annex of IR 2023/138

For the Mobility HDV, the Transport Network INSPIRE Data Theme is listed in section 6.1of the Annex of the IR.

For Meteorology HDV, it could be different, as  the text of the IR 2023/138 has more details (is more strict) than INSPIRE Directive, altrough in article 2 (2) of the IR it is stipulated that the definitions of meteorological data themes are those from INSPIRE Directive. As the IR 2023/138 is more strict for Meteorology  than the INSPIRE Directive, could be a reason why the Meteorology is not listed [here](https://inspire-geoportal.ec.europa.eu/srv/eng/catalog.search#/hvdshome?tab1=tab1)

**Comment by hogredan** [comment]
> @hogredan How is Germany legally interpreting the IR? In all German documentation I read, there is the same list provided as mentioned in the issue description but I cannot find any clear definition anywhere which of the two options I mentioned in the description is true. Or do you maybe have an idea who could comment on the details?

I agree with the opinion of @iuriemaxim. The IR 2023/138 references the entire INSPIRE theme Land Use and doesn't distinguish between the different application schemes. Thus, from my point of view both ELU and PLU data fall under HVD.

**Comment by fabiovinci** [comment]
I also agree with @iuriemaxim and @hogredan.

All the datasets within the scope of the "mentioned" INSPIRE data themes are in the scope of HVD Implementing Regulation.

This also includes all the various application schemas that may exist for a specific data theme.
